src/commands/createWebApp/WebAppCreateStep.ts

```diff
@@ -51,7 +55,51 @@ export class WebAppCreateStep extends AzureWizardExecuteStep<IWebAppWizardContex
         return !context.site;
     }
 
-    private async getAppSettings(context: IWebAppWizardContext): Promise<WebSiteManagementModels.NameValuePair[]> {
+    private getSiteConfig(context: IWebAppWizardContext): WebSiteManagementModels.SiteConfig {
+        const newSiteConfig: WebSiteManagementModels.SiteConfig = {};
+
+        newSiteConfig.appSettings = this.getAppSettings(context);
+
+        const stack: FullWebAppStack = nonNullProp(context, 'newSiteStack');
+        if (context.newSiteOS === WebsiteOS.linux) {
+            newSiteConfig.linuxFxVersion = stack.stack.value === 'java' ?
+                getJavaLinuxRuntime(stack.majorVersion.value, nonNullProp(context, 'newSiteJavaStack').minorVersion) :
+                nonNullProp(stack.minorVersion.stackSettings, 'linuxRuntimeSettings').runtimeVersion;
+        } else {
+            const runtimeVersion: string = nonNullProp(stack.minorVersion.stackSettings, 'windowsRuntimeSettings').runtimeVersion;
+            switch (stack.stack.value) {
+                case 'dotnet':
+                    if (!/core/i.test(stack.minorVersion.displayText)) {
+                        newSiteConfig.netFrameworkVersion = runtimeVersion;
+                    }
```

Comment:
What is this check doing?

Reply:
I'm not sure how familiar you are with the differences between .NET Framework and .NET Core, but this property is only for .NET Framework and doesn't apply to .NET Core. I'm not a huge fan of using `displayText` to figure out which ones are .NET Core (for localization reasons, etc.), but I didn't find a better property in the stack object.